assignment/scripts/function-practice.js

```diff
@@ -50,38 +55,73 @@ console.log( 'isPositive - should say false', isPositive(-3) );
 // 6. Function to return the _last_ item in an array. If the 
 //    array is empty, return `undefined`.
 function getLast( array ) {
-
+  if (array.lentgh = 0) {
```

Comment:
Uh oh.

```suggestion
  if (array.length === 0) {
```

Thankfully, in your case this error will result in an if block that never runs, and it turns out you don't need it. Line 61 will return `undefined` if there are no items in the array.